examples/pytorch/pyg/main.py

```diff
@@ -0,0 +1,188 @@
+"""PyGeo interface training example.
+
+https://github.com/pyg-team/pytorch_geometric/blob/ae84a38f14591ba9b8ce64e704e04ea1271c3b78/examples/graph_sage_unsup_ppi.py#L12
```

Comment:
This comment doesn't seem to be relevant anymore, because you are using Cora.

Reply:
I am using their graphsage model still and most of their training loop.

Reply:
Training loop is the same across many models. Please, remove this comment.